src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -336,33 +301,22 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     };
     rates.targetDownlinkBitrate = this.determineTargetRate();
 
-    // calculate subscribe delay based on bandwidth increasing/decreasing
     const numberOfParticipants = this.subscribedReceiveSet.size();
-    const prevEstimated = this.prevDownlinkStats.bandwidthEstimateKbps;
-    const currEstimated = this.downlinkStats.bandwidthEstimateKbps;
-
-    if (currEstimated > prevEstimated) {
-      // if bw increases, we use recovery delay
-      this.timeBeforeAllowSubscribeMs = this.getSubscribeDelay(
-        NetworkEvent.Increase,
-        numberOfParticipants
-      );
-    } else if (currEstimated < prevEstimated) {
-      // if bw decreases, we use response delay
-      this.timeBeforeAllowSubscribeMs = this.getSubscribeDelay(
-        NetworkEvent.Decrease,
-        numberOfParticipants
-      );
-    }
+    const currentEstimated = this.downlinkStats.bandwidthEstimateKbps;
 
     // If no major changes then don't allow subscribes for the allowed amount of time
```

Comment:
Can we update this comment regarding probeFailed?

Reply:
I took a second look on the comments and updated some. Thanks for bringing it up.